lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,23 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+    tokenized_list = []
+    word = []
+
+    for element in text.lower():
+        if element.isalpha():
+            word.append(element)
+        else:
```

Comment:
add `continue` to the `else` statement and remove `else`

Reply:
removing ‘else’ leads to errors in the checks. why does this happen?

Reply:
I do not know

Reply:
probably, I forgot about continue